File: dayZero/daysConverter.cpp
#include <iostream>
using namespace std;

int main()
{
    int inputDays, years, weeks, days;

    cout << "Enter total days: ";
    cin >> inputDays;

    years = inputDays / 365;
    inputDays = inputDays % 365;
    weeks = inputDays / 7;
    days = inputDays % 7;

    cout << "Years = " << years << endl;
    cout << "Weeks = " << weeks << endl;
    cout << "Days = " << days;

    return 0;
}

File: DAY_0_FOUNDATION/days_converter.py
days=int(input("Enter the total number of days: "))
weeks=days//7
rWeekDays=days%7
years=days//365
rYearDays=days%365
print("The days in week are: ", weeks,"week","And the remaining Days in weeks are: ", rWeekDays)
print("The days in year are: ", years,"year","And the remaining Days in years are: ", rYearDays)